src/lang-go.ts

```diff
@@ -557,12 +558,14 @@ function withFallback<T>({
     fallback: ObservableInput<T>
     delayMilliseconds: number
 }): Observable<T> {
-    return race(
-        of(null).pipe(switchMap(() => from(main))),
+    const mainObservable = from(main).pipe(share())
```

Comment:
@chrismwendt this is the best I could come up with to express "emit from `fallback` after `delay`, unless `main` has emitted, and emit everything from `main`". Happy to take suggestions on this 😄

Reply:
I implemented this independently and came up with exactly the same solution 😺